### DISCUSSION TITLE
saf-training/courses/profile-dev-test/15

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#77] [Ideas]
# saf-training/courses/profile-dev-test/15

Test Kitchen - Getting Started Test Kitchen (http://kitchen.ci) is a robust tool for testing infrastructure code and software on isolated platforms. It provides a consistent, re...

https://mitre.github.io/saf-training/courses/profile-dev-test/15.html

**Comment by seanlongcc** [comment]
While going through the training, I came across some terminology and commands that I didn’t understand or were confused about. It was only after I completed the *Docker Testing Suite* demo that I came across the *Test Kitchen* and *Background & Definition* set of guides. It would be helpful if these were moved up to somewhere before the demos.
